packages/server/src/seeds/searchparameters.ts

```diff
@@ -14,11 +14,13 @@ export async function rebuildR4SearchParameters(): Promise<void> {
 
   const systemRepo = getSystemRepo();
 
+  const promises = [];
   for (const filename of SEARCH_PARAMETER_BUNDLE_FILES) {
     for (const entry of readJson(filename).entry as BundleEntry[]) {
-      await createParameter(systemRepo, entry.resource as SearchParameter);
+      promises.push(createParameter(systemRepo, entry.resource as SearchParameter));
```

Comment:
lol, curious how Postgres handles this

Reply:
We're capped at 10 concurrent connections with the pool, I considered trying to up it and see what happens, but seemed to run fine on my local machine as is.